2021-22/2021-12-02_50p.py

```diff
@@ -29,13 +29,19 @@ def goto(target_x, target_y, reverse):
     delta_y = y2 - y1
     distance = math.sqrt(delta_x**2 + delta_y**2)     # pythagorean theorem
     if ( delta_x ):
-        directio = math.atan(delta_y / delta_x) * 180 / math.pi
+        directio = - math.atan(delta_y / delta_x) * 180 / math.pi
```

Comment:
atan() returns negative values for turning "right", which is positive for the robot